Classes/Command/CacheCommandController.php

```diff
@@ -30,6 +30,13 @@ class CacheCommandController extends CommandController
     protected $cacheService;
 
     /**
+     * Whether the command needs admin access to perform its job
+     *
+     * @var bool
+     */
+    protected $requestAdminPermissions = true;
```

Comment:
In this case it is a bit more complicated than that. The cache commands work very low level, where this is not evaluated.